cmd/contour/serve.go

```diff
@@ -396,13 +396,13 @@ func doServe(log logrus.FieldLogger, ctx *serveContext) error {
 			ResponseHeadersPolicy: &responseHeadersPolicy,
 		},
 	}
-	if clients.ResourcesExist(k8s.GatewayAPIResources()...) {
+	if ctx.Config.GatewayConfig != nil {
 		log.Debug("Gateway API found, adding Gateway API processor")
 		dagProcessors = append(dagProcessors, &dag.GatewayAPIProcessor{
 			FieldLogger: log.WithField("context", "GatewayAPIProcessor"),
 		})
 	} else {
-		log.Debug("Gateway API not found, not adding Gateway API processor")
+		log.Debug("Gateway API, Gateway not configured, not adding Gateway API processor")
```

Comment:
I *think* that if a Gateway is not configured, then we also shouldn't set up the informers (https://github.com/projectcontour/contour/blob/main/cmd/contour/serve.go#L481-L499)

Reply:
👍